src/apply.jl

```diff
@@ -1,14 +1,13 @@
 function ITensors.apply(
   o::ITensor,
   ψ::AbstractITensorNetwork;
-  cutoff=nothing,
-  maxdim=nothing,
   normalize=false,
   ortho=false,
   envs=ITensor[],
   nfullupdatesweeps=10,
   print_fidelity_loss=true,
   envisposdef=false,
+  svd_kwargs...,
```

Comment:
I would call this `apply_kwargs` since it is getting passed to the `apply` function.